examples/tutorials/translations/hindi/Part 06 - Federated Learning on MNIST using a CNN.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>In first line change पाइटोरेक&nbsp; with PyTorch. In 4th <code>एन</code> to <code>n</code>. Fix markdown for all links, proper syntax is [visibleText](actualLink) without any space in between.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done and done!</p>